### DISCUSSION TITLE
Vercel deploy --prebuilt error

### DISCUSSION THREAD

**Discussion by xeriamyuu** [vercel/vercel#8641] [Help] [answered]
Hi, im quite new to coding and i was just wondering if someone could help me figure out why this error is occurring, my vercel build command is running however, when i go to vercel deploy --prebuilt i am met with the following error. Just wondering if anyone could help me in resolving this issue.   Also, ive checked the path which it is looking for and it looks like it does in fact exist? I could be wrong, but any help would be greatly appreciated. 

Thanks!!
![Screenshot 2022-09-27 120726](https://user-images.githubusercontent.com/66210935/192414744-162c8bfc-e6d6-49d6-a67c-bcf81d5907fc.png)
![Untitled](https://user-images.githubusercontent.com/66210935/192414970-8a5a45b1-ca26-4ced-8013-5fc6992a6b48.png)

**Comment by amyegan** [comment]
Thanks for reaching out @xeriamyuu! Sorry you are running into errors. 😞 

If you can share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) or your repository, I am happy to help you find the source of this problem.

**Comment by xeriamyuu** [comment]
Hi,

https://github.com/xeriamyuu/Portfolio

Here is a copy of the repository.


Thanks,
Kevin

**Reply by amyegan** [reply]
Thanks for sharing this Kevin. Are you still getting the same error, or is it a different error now?

Are you seeing any warnings on your local build? I'm not able to build your project locally myself (even after bypassing Sanity). It looks like some pages are trying to call `/api` addresses for static rendering before the project is built. But the code I'm seeing is from a couple of days ago so you have probably made some changes since that time.

If you can share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), then I would be happy to continue investigating the problem.

**Comment by xeriamyuu** [comment, marked as answer]
For some reason, ive changed it to getServerSideProps rather than
getStaticProps and its successfully deployed!!! Thanks for keeping in touch
along the way! :)

Kind Regards,
Kevin Chen
